### DISCUSSION TITLE
Is `StorageError` intentionally not exported?

### DISCUSSION THREAD

**Discussion by c12i** [apalis-dev/apalis#187] [Q&A] [answered]
I was setting up error handling in my application using `apalis` and I could not import `StorageError`, I am wondering whether this is intentional?

this is kind of what I am attempting:

```rust
use apalis::prelude::StorageError;

#[derive(thiserror::Error, Debug)]
pub struct ApiError {
    #[error("Failed to push job to storage: {0:?}")]
    JobQueueError(#[from] StorageError)
}
```

**Comment by geofmureithi** [comment, marked as answer]
It might not be intentional. I can update it in the next release.
Right now you are able to use `JobError` though right?

**Reply by c12i** [reply]
Yes, I can use `JobError`, which I see already implements `From<StorageError>` so I guess this could suffice too

**Reply by geofmureithi** [reply]
It might be better because it covers all possible errors.
